app/main.py

```diff
@@ -1,13 +1,24 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
-    pass
+    return {
+        'line': error['line_number'],
+        'column': error['column_number'],
+        'message': error['text'],
+        'name': error['code'],
+        'source': 'flake8'
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # white your code here
-    pass
+    return{
+        'errors': [format_linter_error(error) for error in errors],
+        'path': file_path,
+        'status': "failed" if [format_linter_error(error) for error in errors]
```

Comment:
```suggestion
        'status': "failed" if errors
```
You can simplify this statement.